ecs-cli/modules/clients/aws/cloudformation/template.go

```diff
@@ -163,9 +163,14 @@ var template = `
     },
     "CreateSecurityGroup": {
       "Fn::Equals": [
-        {
-          "Ref": "SecurityGroup"
-        },
+				{
+					"Fn::Join": [
+						"",
+						{
+							"Ref": "SecurityGroup"
```

Comment:
CreateSecurityGroup should only create one security group, why is "Join" needed here?